lib/rucio/web/rest/flaskapi/v1/rules.py

```diff
@@ -307,13 +298,11 @@ def post(self, rule_id):
         parameters = json_parameters()
         rse_expression = param_get(parameters, 'rse_expression')
         rule_id = param_get(parameters, 'rule_id', default=rule_id)
-        activity = param_get(parameters, 'activity', default=None)
-        source_replica_expression = param_get(parameters, 'source_replica_expression', default=None)
+        override = param_get(parameters, 'override', default=None)
```

Comment:
This alters the API, it's not back compatible. How should we handle it?

Reply:
I believe it’s necessary to accept all three, merge them into `override` if necessary. We should add a comment that this should be removed in the future.

Reply:
Okay yeah, I would do that too.

It should favor the override entry over the old ones if both are speified. I don't think it's necessary to throw an error here.